uint8 chunk_index_1
uint8 chunk_index_2
---
string result
